leptonai/cli/pod.py

```diff
@@ -101,6 +102,15 @@ def pod():
     type=str,
     multiple=True,
 )
+@click.option("--container-image", type=str, help="Container image to run.")
+@click.option(
+    "--container-command",
+    type=str,
+    help=(
+        "Command to run in the container. Your command should listen to the port"
+        " specified by --container-port."
```

Comment:
nit: in pod, we do not have --container-port so the second help sentence can be removed.